test-cases/gemini/gemini-8h-large-num-columns.yaml

```diff
@@ -0,0 +1,30 @@
+test_duration: 500
+n_db_nodes: 3
+n_test_oracle_db_nodes: 1
+n_loaders: 1
+n_monitor_nodes: 1
+instance_type_db: 'i3en.3xlarge'
+instance_type_loader: 'c5.4xlarge'
+
+user_prefix: 'gemini-8h-large-num-columns'
+store_results_in_elasticsearch: False
+
+nemesis_class_name: 'GeminiChaosMonkey'
+nemesis_interval: 5
+
+# gemini
+# cmd: gemini -d -n [NUM_OF_TEST_ITERATIONS] -c [NUM_OF_THREADS] -p [NUM_OF_PARTITION_KEYS_PER_THREAD] -m mixed -f
+# the below cmd runs about 3 hours
+gemini_cmd: "gemini -d --duration 7h --warmup 1h \
+-c 20 -m mixed -f --non-interactive \
+--cql-features normal --async-objects-stabilization-backoff 500ms \
+--replication-strategy \"{'class': 'SimpleStrategy', 'replication_factor': '3'}\" \
+--max-mutation-retries 5 --max-mutation-retries-backoff 500ms \
+--max-partition-keys 12 --min-partition-keys 8 \
+--max-clustering-keys 20 --min-clustering-keys 12 \
+--max-columns 100 --min-columns 80 --verbose"
```

Comment:
I'm thinking if it's better to test many columns or simply large partitions where each partition has many clustering keys.
The clustering keys has more. affect on scylla than how many columns a row has.

On the other hand, maybe the mix of them is worth testing.

Do we know how big each column? what is the total size of 1 partition?

Reply:
Unfortunately, it is difficult to predict total size of partition, because gemini during could alter the table, and not only insert but also delete the rows. and each run it generate new schema.

Reply:
From what I understands now it's not the number of rows, but the number of columns.
So ignore my comment.